tests/annotations/tests.py

```diff
@@ -1017,6 +1017,14 @@ def test_annotation_and_alias_filter_related_in_subquery(self):
             ],
         )
 
+    def test_annotation_exists_empty_query_result(self):
+        self.assertIs(
+            False,
+            Company.objects.annotate(exists=Exists(Company.objects.none()))
+            .filter()
+            .exists(),
+        )
+
```

Comment:
@rajdesai24 as explained in the previous comment this is not a proper regression test because it passes without the `Exists.empty_result_set_value = False` change.

You want to be testing the value of the `exists` annotation here and not the `.exists()` method itself.

Moreover the empty `filter()` call is noop; it doesn't do anything.

Refer to the objects created in `setUpTestData` to craft a query that matches a single object and can be retrieved by `get()` and assert against the value of the returned object's `exists` annotation.